README.md

```diff
@@ -235,23 +235,37 @@ Batching is particularly useful with GraphQL. Using such techniques as preloadin
 Let's take a look at the simple [graphql-ruby](https://github.com/rmosolgo/graphql-ruby) schema example:
 
 ```ruby
-Schema = GraphQL::Schema.define do
-  query QueryType
+class MyprojectSchema < GraphQL::Schema
```

Comment:
could we make the naming a bit more consistent`MyProjectSchema` or simply `Schema`

Reply:
I like to use names that are really obviously not a 'magic' name, but something that can be changed as desired.